framework/platform/bpl/uci/cfg/bpl_cfg_helper.cpp

```diff
@@ -103,7 +106,7 @@ int cfg_get_channel(int index, int *channel)
     return RETURN_OK;
 }
 
-int cfg_get_wep_key(int index, int keyIndex, char *key)
+int cfg_get_wep_key(const std::string &interface_name, int keyIndex, char *key)
```

Comment:
I think this whole function can and should be removed. WEP is obsolete and we don't support it.

Reply:
But lets not make this PR block on that...

Reply:
Right, forgot to mark this as (nitpick)

Reply:
ok - we can do that as part of #805